### PR TITLE
2022q3 Changelog.

### PR DESCRIPTION
**Pull request by tzarc**
## Description

Let's get this branch merged.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/ChangeLog/20220827.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+### Generic wear-leveling for EEPROM emulation ([#16996](https://github.com/qmk/qmk_firmware/pull/16996), [#17376](https://github.com/qmk/qmk_firmware/pull/17376), [#18102](https://github.com/qmk/qmk_firmware/pull/18102)) :id=wear-leveling
+
+QMK has had the ability to write to internal MCU flash in order to emulate EEPROM for some time now, but it was only limited to a small number of MCUs. The base HAL used by QMK for a large number of ARM devices provides a "proper" embedded MCU flash driver, so _@tzarc_ decoupled the wear-leveling algorithm from the old flash writing code, improved it, wrote some tests, and enabled its use for a much larger number of other devices... including RP2040's XIP flash, and external SPI NOR Flash.
+
+See the [EEPROM Driver](/eeprom_driver.md) documentation for more information.
+
+---
```
Comment (drashna) [inline docs/ChangeLog/20220827.md:50]: Could we get mouse/pointing device stuff added here?  There were a lot of changes. 

* Add support for large Mouse Reports ([#16371](https://github.com/qmk/qmk_firmware/pull/16371))
* Improve PS/2 mouse performance ([#17111](https://github.com/qmk/qmk_firmware/pull/17111))
* Mouse key kinetic mode fix ([#17176](https://github.com/qmk/qmk_firmware/pull/17176))
* Cirque trackpad features: circular scroll, inertial cursor ([#17482](https://github.com/qmk/qmk_firmware/pull/17482))
  * Says cirque but this was both
* Various Cirque Pinanacle trackpad improvements.
* Create generic Pointing Device Pin defines ([#17776](https://github.com/qmk/qmk_firmware/pull/17776))
* PMW33XX drivers overhaul ([#17613](https://github.com/qmk/qmk_firmware/pull/17613))

File: docs/ChangeLog/20220827.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+QMK _finally_ picked up support for RP2040-based boards, such as the Raspberry Pi Pico, the Sparkfun Pro Micro RP2040, and the Adafruit KB2040. One of QMK's newest collaborators, _@KarlK90_, effectively did `/micdrop` with RP2040, with a massive set of changes to both QMK and the repository QMK uses for the base platform support, ChibiOS[-Contrib]. There has been a flurry of development this breaking changes cycle related to RP2040 from a large number of contributors -- so much so that almost all standard QMK hardware subsystems are supported.
+
+Check the [RP2040 platform development page](platformdev_rp2040.md) for all supported peripherals and other hardware implementation details.
+
+### Teaching the CLI to flash binaries ([#16584](https://github.com/qmk/qmk_firmware/pull/16584)) :id=cli-flash-binaries
```
Comment (spidey3) [inline docs/ChangeLog/20220827.md:11]: While "Teaching the CLI to flash..." makes sense to us, it's somewhat of an odd linguistic construction.
Perhaps this should be renamed to "Enable flashing pre-built firmware binaries using QMK CLI" or something like that.

**Review by spidey3 — COMMENTED** [review]
One minor comment on language...

File: docs/ChangeLog/20220827.md (hunk 2 above)
Comment (tzarc) [reply docs/ChangeLog/20220827.md:11]: That was the name of the original PR, but can change it in the docs, sure.

**Review by noroadsleft — APPROVED** [review]

**Pull request merged**